projects/policekillings/package.json

```diff
@@ -1,14 +1,16 @@
 {
-  "name": "police_killings_dashboard",
+  "name": "project-one",
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 10001",
```

Comment:
`-p XXXXX` sets what port this will run on, and I made the test project ones unique so they could be run concurrently if needed. 10001 and 10002 are in use, so recommend choosing a value specific to this project with that in mind.

Reply:
Oh okay. Got it. I learned something new today. Thanks!